index.html

```diff
@@ -2324,19 +2324,19 @@ <h4>Interaction Protocols Response</h4>
       </section>
 
       <section>
-        <h4>`website` Interaction Protocol</h4>
+        <h4>`inviteRequest` Interaction Protocol</h4>
         <p>
-The `website` interaction protocol is used by a local system to signal to the
-remote system that it would like to redirect the remote system to a specific
-URL, such as a website where an individual can engage in a use-case specific
-interaction. If the `website` interaction protocol is selected, the local
-system sends data using an HTTP POST to instruct the remote system where to
-send the individual. An example of the POST data is shown below:
+The `inviteRequest` interaction protocol is used by a local system to signal to
+the remote system that it would like an invitation to a remote system via a
+specific URL, such as a website where an individual can engage in a use-case
+specific interaction. If the `inviteRequest` interaction protocol is selected,
+the local system sends data using an HTTP POST to instruct the remote system
+where to send the individual. An example of the POST data is shown below:
         </p>
 
-        <pre class="example nohighlight" title="A website interaction protocol redirect request">
+        <pre class="example nohighlight" title="A response to an invitation request">
 {
-  "url": "https://website.example/interactions/8372974",
+  "url": "https://website.example/transactions/8372974",
```

Comment:
Hmm, I'd like to avoid reintroducing "transactions" anywhere in the spec to avoid confusion, though I realize this could really be any value for this particular case. How about "checkouts"?

```suggestion
  "url": "https://website.example/checkouts/8372974",
```

Reply:
I'd be ok with "invitation" -- I'd just be worried that people would think it then has to be "invitation", which is why having something different (or perhaps just two different examples?) would be helpful.